draft-ietf-scitt-architecture.md

```diff
@@ -704,8 +704,8 @@ Unprotected_Header = {
 
 Receipts are based on COSE Signed Merkle Tree Proofs ({{-COMETRE}}) with an additional wrapper structure that adds the following information:
 
-- version: Receipt version number; this should be set to `0` for implementation of this document. We envision that future version of SCITT may add support for more complex receipts; for instance, registrations on multiple TS, receipts for dependency graphs and endorsements of Signed Claims, etc.
-- ts_identifier: The DID of the Transparency Service that issued the claim. Verifiers MAY use this DID as a key discovery mechanism to verify the COSE Merkle Root signature; in this case the verification is the same as for Signed Claims and the signer should include the Key ID header. Verifiers MUST support the `did:web` method, all other methods are optional.
+- version: Receipt version number; MUST be set to `0` for implementation of this document. We envision that future version of SCITT may add support for more complex receipts; for instance, registrations on multiple TS, receipts for dependency graphs and endorsements of Signed Claims, etc.
```

Comment:
```suggestion
- version: Receipt version number; MUST be set to `0` for implementation of this document.
```

Reply:
Everyone understands what the purpose of version numbers are. So, there is no need to talk about what could be done in the future to create a new version

Reply:
+1